### PR TITLE
ENH: Provide models with dropout

### PR DESCRIPTION
**Pull request by Leengit**
Add one model each for tensorflow and pytorch to include a dropout layer.  This is for use with superpixel feature vectors.

### CODE REVIEW

**Comment by Leengit** [comment]
These file changes build upon those of Pull Request #22.  That pull request should be merged before this one.
